packages/wallets/src/components/CreatePassword/CreatePassword.tsx

```diff
@@ -1,28 +1,47 @@
 import React from 'react';
+import { useAvailableMT5Accounts, useCreateOtherCFDAccount } from '@deriv/api';
 import PasswordShowIcon from '../../public/images/ic-password-show.svg';
 import './CreatePassword.scss';
 
+type TPlatformMT5 = NonNullable<ReturnType<typeof useAvailableMT5Accounts>['data']>[number]['platform'];
+
+type TPlatformOtherAccounts =
+    | Parameters<NonNullable<ReturnType<typeof useCreateOtherCFDAccount>['mutate']>>[0]['payload']['platform'];
+
+type TPlatform = TPlatformMT5 | TPlatformOtherAccounts;
+
+const platformToTitleMapper: Record<TPlatform, string> = {
+    ctrader: 'cTrader',
+    derivez: 'Deriv EZ',
+    dxtrade: 'Deriv X',
+    mt5: 'Deriv MT5',
+};
+
 type TProps = {
     icon: React.ReactNode;
     onPasswordChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
     onPrimaryClick: () => void;
-    platform: string;
+    platform: TPlatform;
 };
 
 const CreatePassword: React.FC<TProps> = ({ icon, onPasswordChange, onPrimaryClick, platform }) => {
     return (
         <div className='wallets-create-password'>
             {icon}
-            <div className='wallets-create-password-title'>Create a {platform} password</div>
+            <div className='wallets-create-password-title'>Create a {platformToTitleMapper[platform]} password</div>
```

Comment:
```suggestion
            <div className='wallets-create-password-title'>Create a {platform_name} password</div>
```

This can be done once before the render 
```
const platform_name = platformToTitleMapper[platform];
```